apps/api/prisma/schema.prisma

```diff
@@ -55,6 +56,17 @@ model QuestionVote {
   @@id([userId, questionId])
 }
 
+model QuestionsAnswers {
+  id         Int      @id @default(autoincrement())
+  userId     Int      @map("_userId")
```

Comment:
Powinniśmy to zmienić na coś bardziej jednoznacznego, tak samo jak w pytaniach - np createdById